### DISCUSSION TITLE
[Error] GetDocument failed:

### DISCUSSION THREAD

**Discussion by rdelphin** [tinacms/tinacms#4219] [Help]
Hi Can anyone help with this error? I use to have forestry now moving to Tina.

[Error] GetDocument failed: Unable to fetch, errors: Unable to find collection for file at content\xxxxxx.md

**Comment by kldavis4** [comment]
Can you post your tina config?

**Reply by rdelphin** [reply]
import { defineConfig } from "tinacms";

// Your hosting provider likely exposes this as an environment variable
const branch = process.env.HEAD || process.env.VERCEL_GIT_COMMIT_REF || "main";

export default defineConfig({
  branch,
  clientId: "cd73b96d-a2b2-4010-b1e8-946bef75277f", // Get this from tina.io
  token: "3a81e1640a9b3082be7ad1fe8566b300b41bcee6", // Get this from tina.io
  client: { skip: true },
  build: {
    outputFolder: "admin",
    publicFolder: "public",
  },
  media: {
    tina: {
      mediaRoot: "",
      publicFolder: "public",
    },
  },
  schema: {
    collections: [
      {
        format: "md",
        label: "Events",
        name: "events",
        path: "content/events",
        match: {
          include: "**/*",
        },
        fields: [
          {
            type: "rich-text",
            name: "body",
            label: "Body of Document",
            description: "This is the markdown body",
            isBody: true,
          },
          ...eventsFields(),
        ],
      },
      {
        format: "md",
        label: "Projects",
        name: "projects",
        path: "content/projects",
        match: {
          include: "**/*",
        },
        fields: [
          {
            type: "rich-text",
            name: "body",
            label: "Body of Document",
            description: "This is the markdown body",
            isBody: true,
          },
          ...projectsFields(),
        ],
      },
      {
        format: "md",
        label: "Directors",
        name: "directors",
        path: "content/directors",
        match: {
          include: "**/*",
        },
        fields: [
          {
            type: "rich-text",
            name: "body",
            label: "Body of Document",
            description: "This is the markdown body",
            isBody: true,
          },
          ...directorsFields(),
        ],
      },
      {
        format: "md",
        label: "Posts",
        name: "posts",
        path: "content/posts",
        match: {
          include: "**/*",
        },
        fields: [
          {
            type: "rich-text",
            name: "body",
            label: "Body of Document",
            description: "This is the markdown body",
            isBody: true,
          },
          ...postFields(),
        ],
      },
      {
        format: "md",
        label: "Pages",
        name: "pages",
        path: "content",
        match: {
          include: "*",
        },
        fields: [
          {
            type: "rich-text",
            name: "body",
            label: "Body of Document",
            description: "This is the markdown body",
            isBody: true,
          },
          ...pageFields(),
        ],
      },
      {
        format: "toml",
        label: "Config",
        name: "config",
        path: ".",
        ui: {
          allowedActions: {
            create: false,
            delete: false,
          },
        },
        match: {
          include: "config",
        },
        fields: [
          {
            name: "dummy",
            label: "Dummy field",
            type: "string",
            description:
              "This is a dummy field, please replace it with the fields you want to edit. See https://tina.io/docs/schema/ for more info",
          },
        ],
      },
    ],
  },
});

**Reply by kldavis4** [reply]
If you are on windows it is probably this bug: https://github.com/tinacms/tinacms/issues/4210

**Reply by rdelphin** [reply]
Yes. I'm on windows. Should I replace my Collections

collections: [
      {
        format: "md",
        label: "Events",
        name: "events",
        path: "content/events",
        match: {
          include: "**/*",
        },
        fields: [
          {
            type: "rich-text",
            name: "body",
            label: "Body of Document",
            description: "This is the markdown body",
            isBody: true,
          },
          ...eventsFields(),
        ],
      },

To the one in #4219

collections: [
{
  name: "pages",
  path: "content",
  match: {
    include: "*",
  }
},
{
  name: "posts",
  path: "content/posts",
  match: {
    include: "**/*",
  }
},
{
  name: "authors",
  name: "content/authors",
  match: {
    include: "**/*",
  }
}

**Reply by kldavis4** [reply]
you don't need the match property unless you need to filter the files under the specified path, so I would start by removing that from wherever it is not needed

**Reply by rdelphin** [reply]
Not sure what to delete. I'm new to Tina.

**Reply by rdelphin** [reply]
Any help?

**Reply by kldavis4** [reply]
Here is the relevant documentation: https://tina.io/docs/reference/collections/#matchinclude
